pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "great_expectations_cloud"
-version = "0.0.3dev5"
```

Comment:
Explicitly adding the `.` between the `micro` and `pre-release` components.
https://pypi.org/project/great-expectations-cloud/0.0.3.dev5/